### ISSUE TITLE
Connecting to Colab

### ISSUE THREAD

**Issue by Nelson-Gon** [lmb-freiburg/Unet-Segmentation#70]
Hello,

I'm currently running on a 4GB Nvidia GPU and fine tuning takes hours, sometimes failing with "Out of GPU memory". I would like to know if there is a way I can connect the plugin to Colab maybe by using a colab notebook as the remote host which would perhaps save me some time. I would like to use Colab since it would give me some free GPU time unlike AWS.

Thank you

**Comment by ThorstenFalk** [comment]
Hi,

I have no experience with Colab, but as far as I understand Colab is only a remote python notebook. If you cannot install the caffe_unet backend on a Colab server I see no way of using Colab notebooks with the UNet Segmentation plugin. You're better off using tensorflow or pytorch in conjunction with ImgLib2 from ImageJ directly if you want to use Colab notebooks. But this means basically rewriting the whole functionality of data normalization, augmentation, tiled prediction, .... So if you are not very experienced, it is no option, I fear.

**Comment by Nelson-Gon** [comment]
Alright, thank you very much.
